src/ontology/mondo-ingest.Makefile

```diff
@@ -422,6 +426,9 @@ $(MAPPINGSDIR)/rejected-mappings.tsv:
 $(MAPPINGSDIR)/rejected-mappings-sssom.tsv: $(MAPPINGSDIR)/rejected-mappings.tsv
 	sssom parse $< -m metadata/mondo.sssom.config.yml --no-strict-clean-prefixes -o $@
 
+tmp/mondo-ingest.owl: mondo-ingest.owl
+	cp $< $@
```

Comment:
This was the source of the problem.

1. The sync pipeline depends on tmp/mondo-ingest.db
2. tmp/mondo-ingest.db depends ofn tmp/mondo-ingest.owl
3. There was no goal for tmp/mondo-ingest.owl

Reply:
Ugh, I am thinking about this and I thought this was something I'd addressed. But perhaps what I changed was actually moving `merged.db` to `tmp/merged.db`. 

Sorry I never caught this!

Do I need to update my development workflow to the following?
Make sure code works in the following ways, in order.
1. Debugging in my IDE
2. `sh run.sh make ...`
3. `sh run.sh make ... -B`
4. `build-mondo-ingest`

I've only been doing (1) and (2). (4) is something we're all thinking about making easier, but I don't think we can expect to do 100% of the time. Should I be adding (3) to my workflow? Would that have caught this problem? If so, do I do (3) and just never need to do (2)?

Reply:
At some point I'd include `sh run.sh make ... -B` so that everything is rebuilt, but I am also wondering how versions of all targets are being captured between runs to know that the inputs are the same from one run to the next.